### PR TITLE
Update the README.md file to describe the repo. Add my portfolio.

### PR DESCRIPTION
**Pull request by yueyue200830**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -4,15 +4,113 @@
...
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <div id="about">
+        <h2>About Me</h2>
+        <p>
+          I'm Jiayi, an undergraduate student majored in CS. 
+          I got in touch with computer when I was in kindergarden, playing both online and offline computer games.
+          After growing up, I found that computer can make life and work more efficent, just like mechines in 19th century.
```
Comment (akovski) [inline portfolio/src/main/webapp/index.html:18]: s/mechines/machines

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -4,15 +4,113 @@
...
+          Besides, making tools for use and customing my own computer is really interesing to me.
```
Comment (akovski) [inline portfolio/src/main/webapp/index.html:19]: s/customing/customizing

**Comment by yueyue200830** [comment]
> do you have url for this webpage?

Sure. https://jzhu-sps-summer20.df.r.appspot.com/

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (yueyue200830) [reply portfolio/src/main/webapp/index.html]: Thanks! Fixed.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (yueyue200830) [reply portfolio/src/main/webapp/index.html]: Fixed.

**Comment by akovski** [comment]
Cute cat!

**Pull request merged**